docs/user/bots.rst

```diff
@@ -1908,6 +1908,26 @@ Use this command to create/update the database and reload the bot:
 
 .. _intelmq.bots.experts.deduplicator.expert:
 
+Domain valid
+^^^^^^^^^^^^
+
+**Information**
+
+* `name:` `intelmq.bots.experts.domain_valid.expert`
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `description:` Valid first level domain from TLD list `tlds-alpha-by-domain.txt`. Latest list: https://data.iana.org/TLD/
+
+**Configuration Parameters**
+   * `domain_field`- event domain field
```

Comment:
```suggestion

- `domain_field`: The name of the field to be validated.
```
The empty line before the list is important, otherwise the list is not properly formatted.